gulpfile.esm.js

```diff
@@ -83,21 +81,22 @@ function cordovaPrepare() {
 
 function xcode() {
   return runCommand(
-      (platform === 'ios' || platform === 'osx') ?
-          `rsync -avc apple/xcode/${platform}/ platforms/${platform}/` :
-          'echo not running on apple, skipping xcode rsync');
+    platform === "ios" || platform === "osx"
+      ? `rsync -avc apple/xcode/${platform}/ platforms/${platform}/`
+      : "echo not running on apple, skipping xcode rsync"
+  );
 }
 
 function cordovaCompile() {
-  const platformArgs = platform === 'android' ? '--gradleArg=-PcdvBuildMultipleApks=true' : '';
-  const compileArgs = platform === 'ios' ? '--device' : '';
-  let releaseArgs = '';
-  if (isRelease) {
-    releaseArgs = '--release';
-    if (platform === 'android') {
-      releaseArgs += ` -- --keystore=${args.KEYSTORE} --storePassword=${args.STOREPASS} --alias=${
-          args.KEYALIAS} --password=${args.KEYPASS}`;
-    }
+  const platformArgs = platform === "android" ? "--gradleArg=-PcdvBuildMultipleApks=true" : "";
+  // Use flag -UseModernBuildSystem=0 as a workaround for Xcode 10 compatibility until upgrading to
```

Comment:
This has been removed by @jyyi1. Please update.

Reply:
Sorry, not exactly sure what you mean - remove  line 91? or the comment

Reply:
See https://github.com/Jigsaw-Code/outline-client/pull/1147. Your change is undoing that.

Reply:
Ah, it's just the comment. removed. i added a back a build flag intentionally, i'm trying to get it to work in the CI

Reply:
ah, it was just the comment. i added in a build arg of my own in my attempts to get off of fastlane on CI